meet-data/nasa/1613/1609/1604/check_lifters

```diff
@@ -0,0 +1 @@
+Summary: 0 errors, 0 warnings.
```

Comment:
This file looks like an error.

Reply:
@sstangl Yes the entire subfolder 1609 should be deleted. What is the best way to do this so that I don't undo the changes you made to my commit?